cmd/installer/pkg/iso.go

```diff
@@ -98,6 +98,18 @@ func CreateUKIISO(iso, dir, arch string) error {
 		return err
 	}
 
+	if _, err := cmd.Run("mmd", "-i", efiBootImg, "::loader"); err != nil {
+		return err
+	}
+
+	if _, err := cmd.Run("mmd", "-i", efiBootImg, "::loader/keys"); err != nil {
+		return err
+	}
+
+	if _, err := cmd.Run("mmd", "-i", efiBootImg, "::loader/keys/SECBOOT"); err != nil {
```

Comment:
`SECBOOT` here is arbitrary, just a name for the key group. Could be made user-configurable maybe?

Reply:
maybe just `auto`?

Reply:
```NAME being the display name for the set of variables in the menu. If one of the sets is named **auto** then it might be enrolled automatically depending on whether "secure-boot-enroll" is set to force or not.```

i gues it then auto-enrolls

Reply:
ooooo nice